Source/Scene/BingMapsImageryProvider.js

```diff
@@ -110,7 +110,8 @@ define([
         this._culture = defaultValue(options.culture, '');
         this._tileDiscardPolicy = options.tileDiscardPolicy;
         this._proxy = options.proxy;
-        this._credit = new Credit('<a href="http://www.bing.com"><img src="' + BingMapsImageryProvider._logoData + '" title="Bing Imagery"/></a>');
+        var logoUrl = buildModuleUrl('Assets/Images/bing_maps_credit.png');
```

Comment:
Can we set this string as a static variable?  It's helpful if you need to switch out the logo for the dark or light version.

Reply:
Updated.  I made it public API as well.  I assume previously you were cheating and relying on private undocumented functionality to change the logo.